deploy/kubernetes-1.13/pmem-unified-csi-direct.yaml

```diff
@@ -0,0 +1,77 @@
+# Deploys the pmem-csi driver onto exactly one node, running in unified mode.
+# That CSI drivers gets exposed with a NodePort service and the fixed port
+# 30123 for use with csi-test sanity testing.
```

Comment:
Just curious to konw, where are we doing this? i mean fixed port.

Reply:
I have no idea. This text is in pmem-unified-csi-direct.yaml only because this file is based on pmem-unified-csi-lvm.yaml